src/Commands/MachineToken/DeleteCommand.php

```diff
@@ -15,14 +16,27 @@ class DeleteCommand extends TerminusCommand
      * Removes a machine token from the logged-in user's account
      *
      * @name machine-token:delete
+     * @param string $machine_token_id The ID of the machine token to be deleted
+     * @throws \Terminus\Exceptions\TerminusException
      * @aliases mt:delete
      *
-     * @param machine_token_id The ID of the machine token to be deleted
      * @usage terminus machine-token:delete <machine_token_id>
      *   Removes the given machine token from the user's account
      */
     public function delete($machine_token_id) {
+        $user = $this->session()->getUser();
 
+        // Find the token. Will throw an exception if it doesn't exist.
+        $machine_token = $user->machine_tokens->get($machine_token_id);
+        $name = $machine_token->get('device_name');
+
+        $this->log()->notice('Deleting {token} ...', ['token' => $name]);
+        $response = $machine_token->delete();
+        if ($response['status_code'] == 200) {
```

Comment:
By "break the old stuff" I mean "require a change to the old stuff" I woudn't leave the existing code in a broken state obviously